cfme/configure/settings.py

```diff
@@ -62,26 +62,29 @@ def __init__(self, description=None, scope=None, days=None, hours=None, timezone
         self.hours = hours
         self.timezone = timezone
 
-    def create(self):
+    def create(self, negative=False):
```

Comment:
I see what you mean, when cancel is True the form cannot be submitted.  I think it would be good to keep consistent language in the page models, even if a 'cancel' button isn't necessarily being clicked.